x/gov/keeper/keeper.go

```diff
@@ -181,6 +190,28 @@ func (k Keeper) ModuleAccountAddress() sdk.AccAddress {
 	return k.authKeeper.GetModuleAddress(types.ModuleName)
 }
 
+func (k Keeper) validateProposalLengths(metadata, title, summary string) error {
+    if err := k.assertMetadataLength(metadata); err != nil {
+        return err
+    }
+    if err := k.assertSummaryLength(summary); err != nil {
+        return err
+    }
+    if err := k.assertTitleLength(title); err != nil {
+        return err
+    }
+    return nil
+}
+
+// assertTitleLength returns an error if given title length
+// is greater than a pre-defined MaxTitleLen.
+func (k Keeper) assertTitleLength(title string) error {
+	if title != "" && uint64(len(title)) > k.config.MaxTitleLen {
+		return types.ErrMetadataTooLong.Wrapf("got title with length %d", len(title))
```

Comment:
Note to myself, in the backport PR, make this message identical as the metadata message.

Reply:
Nit, for main let's create a new error for the title being too long. I'll revert in the backport.